olcUTIL_Geometry2D.h

```diff
@@ -1583,7 +1583,8 @@ namespace olc::utils::geom2d
 	template<typename T1, typename T2>
 	inline constexpr bool contains(const circle<T1>& c1, const circle<T2>& c2)
 	{
-		return (c1.pos - c2.pos).mag2() <= (c1.radius - c2.radius) * (c1.radius - c2.radius);
+		return (std::sqrt(std::pow(c2.pos.x - c1.pos.x, 2) + std::pow(c2.pos.y - c1.pos.y, 2)) + c2.radius) <= c1.radius;

```

Comment:
`std::pow` is extremely slow; especially so for simple integer exponents like 2. Just write it out by hand. See https://baptiste-wicht.com/posts/2017/09/cpp11-performance-tip-when-to-use-std-pow.html

Reply:
does the compiler not catch this?

Reply:
Not one I've seen; particularly since implementers can simply say "Because the Standard Says So"<sup>tm</sup> and leave it at that.

Reply:
well for now, its more important that it works, so propose as issue, or pr after i merge this